packages/api-request-builder/src/query-id.js

```diff
@@ -6,7 +6,7 @@
  * @throws If `id` is missing.
  * @return {Object} The instance of the service, can be chained.
  */
-export function byId(id: string): Object {
+export function byId(id: string): { [key: string]: any } {
```

Comment:
here i would assume the return types is a request, i think if that's the case it would make sense to point out Request as a return type.

Reply:
Not really, `Object` is actually just an alias for `any` in Flow.

Don't worry much about this, when we migrate the packages to TS, we will fix all the current "broken" types.